src/library/Uno.Themes.WinUI.Markup/ResourceValue.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.MarkupHelpers;
+
+
+namespace Uno.Themes.Markup
+{
+	public struct ResourceValue<T>
+	{
+		public ResourceValue(string key, bool isThemeResource = true)
```

Comment:
I think it might be better to default `isThemeResource` to `false`. I know it'd mean we'd have more constructor calls with `isThemeResource: true` compared to only having `isThemeResource: false` for the `Styles` class but I think that `StaticResource` makes more sense as the default resource retrieval markup extension here

Reply:
It's okay, I changed